docs/guide/testing.md

```diff
@@ -199,3 +199,44 @@ To use component previews:
 # config/application.rb
 config.view_component.preview_paths << "#{Rails.root}/spec/components/previews"
 ```
+
+## Testing Interactive Components
```

Comment:
@joelhawksley added this documentation. Let me know if this is unclear. Thank you!